### PR TITLE
Docs: Update cli templates for v7 beta

### PR DESCRIPTION
**Pull request by kasperpeulen**
Issue:

## What I did

- Only one comment about args
- Only one comment above meta
- Change links to point to 7.0

### CODE REVIEW

File: code/renderers/vue3/template/cli/Page.stories.js [hunk 1]
```diff
@@ -4,21 +4,15 @@ import MyPage from './Page.vue';
 export default {
   title: 'Example/Page',
   component: MyPage,
```
Comment (jonniebigodes) [inline code/renderers/vue3/template/cli/Page.stories.js:6]: @kasperpeulen seems that you accidentally removed the `render` function am I correct?

**Review by jonniebigodes — CHANGES_REQUESTED** [review]
@kasperpeulen, this is shaping up great. Thanks for taking care of this so promptly. Appreciate it. From my end, I left a couple of items that require addressing, and this should be good to go. I'll defer to @shilman's input so we can merge this.

File: code/frameworks/angular/template/cli/Header.stories.ts [hunk 2]
```diff
@@ -18,7 +18,7 @@ const meta: Meta<Header> = {
     }),
   ],
   parameters: {
-    // More on Story layout: https://storybook.js.org/docs/angular/configure/story-layout
+    // More on Story layout: https://storybook.js.org/docs/7.0/angular/configure/story-layout
```
[the remark this answers is not in the document]
Comment (kasperpeulen) [reply code/frameworks/angular/template/cli/Header.stories.ts:21]: Sounds good to me.

File: code/renderers/react/template/cli/ts/Button.stories.ts [hunk 3]
```diff
@@ -2,13 +2,16 @@ import type { Meta, StoryObj } from '@storybook/react';
 
 import { Button } from './Button';
 
-// More on how to set up stories at: https://storybook.js.org/docs/react/writing-stories/introduction#default-export
+// TODO
+// 1. Only one comment about args <- For sure
+// 2. Only one comment above meta
+// 3. And change links to point to 7.0
```
[the remark this answers is not in the document]
Comment (kasperpeulen) [reply code/renderers/react/template/cli/ts/Button.stories.ts:8]: Will fix that :D

File: code/renderers/vue3/template/cli/Button.stories.js [hunk 4]
```diff
@@ -1,27 +1,10 @@
 import MyButton from './Button.vue';
 
-// More on default export: https://storybook.js.org/docs/vue/writing-stories/introduction#default-export
+// More on how to set up stories at: https://storybook.js.org/docs/7.0/vue/writing-stories/introduction
 export default {
   title: 'Example/Button',
   component: MyButton,
-  // This component will have an automatically generated docsPage entry: https://storybook.js.org/docs/vue/writing-docs/docs-page
   tags: ['docsPage'],
```
[the remark this answers is not in the document]
Comment (kasperpeulen) [reply code/renderers/vue3/template/cli/Button.stories.js:7]: @jonniebigodes I would say that now that we have default render functions for all major renderers, we should only show the render function when it is different than the default. What do you think @shilman ?

File: code/frameworks/ember/template/cli/1-Button.stories.js [hunk 5]
```diff
@@ -2,20 +2,19 @@ import { hbs } from 'ember-cli-htmlbars';
 import { action } from '@storybook/addon-actions';
 import { linkTo } from '@storybook/addon-links';
 
-// More on default export: https://storybook.js.org/docs/ember/writing-stories/introduction#default-export
+// More on how to set up stories at: https://storybook.js.org/docs/7.0/ember/writing-stories/introduction
```
[the remark this answers is not in the document]
Comment (kasperpeulen) [reply code/frameworks/ember/template/cli/1-Button.stories.js:5]: I think so too, I will change it.

File: code/renderers/vue3/template/cli/Button.stories.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (jonniebigodes) [reply code/renderers/vue3/template/cli/Button.stories.js:7]: Thanks for the clarification on this @kasperpeulen !  And we can add an example featuring a `render` function further down the line as we previously discussed.

**Review by jonniebigodes — APPROVED** [review]
@kasperpeulen all is good on my end. Thanks for providing the clarification and addressing the feedback so promptly. Appreciate it 🙏 ! Merge when able

**Pull request merged**